modules/managers/galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/test/java/dev/galasa/zos3270/datastream/InboundTest.java

```diff
@@ -131,4 +131,40 @@ public void testSscpLuDataStreamEndingWithNewlinePutsCursorInCorrectPosition() t
         assertThat(screenStr).contains("WELCOME TO SIMBANK", "\n");
         assertThat(cursorPosition).isEqualTo(320);
     }
+
+    @Test
+    public void testCanRenderScreenWithBadModifyFieldOrder() throws Exception {
+
+        // Given...
+        Charset codePage = Charset.forName("1047");
+
+        String inbound3270Header = "0000000000";
+
+        // Create a bad modify field order with invalid attributes
+        // This one says it has 81 attribute type/value pairs, but there are only three
+        // and all three are not known attribute types
+        String badModifyField = "2c51db301106e813";
```

Comment:
This does not look like the bad modify field that was sent to us in the story, it looks fine to me just numbers and letters, so would be good to hear why this is a bad modify field :)